wallmatrix/sources/mbta.py

```diff
@@ -7,18 +8,28 @@
 from wallmatrix.sources import Source
 
 
+class Line(TypedDict):
```

Comment:
I think this deserves a more descriptive name: `TransitLine` or `SubwayLine`? Someone skimming through might otherwise think it's referring to drawing lines on the image.

Not currently sure if the implementation is generic over subway/bus (although maybe it should be, given we live close to the 1...)